### PR TITLE
[KAFKA] Add basic sanity and placement shakedown tests

### PR DESCRIPTION
**Pull request by mbdcos**
Migrate the following from dcos-kafka-service/integration/tests/
* test_sanity.py
* test_placement.py

### CODE REVIEW

File: frameworks/kafka/tests/test_utils.py [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+        except dcos.errors.DCOSHTTPException:
+            return []
+
+    def success_predicate(tasks):
+        return len(tasks) == 1, 'Failed to get task'
+
+    return spin.spin(fn, success_predicate)
+
+
+def task_id_changes(broker_name, task_id):
```
Comment (susanxhuynh) [inline frameworks/kafka/tests/test_utils.py]: You might want to remove this and use the existing function check_tasks_updated(): https://github.com/mesosphere/dcos-commons/blob/master/testing/sdk_tasks.py#L37

File: frameworks/kafka/tests/test_sanity.py [hunk 2]
```diff
@@ -1,25 +1,250 @@
...
+    def fun():
+        return service_cli('endpoints broker')
+
+    def ends(result):
+        return (
+            len(result['native']) == DEFAULT_BROKER_COUNT,
+            'waiting for the expected number of brokers to come online'
+        )
+
+    address = spin.spin(fun, ends)
```
Comment (susanxhuynh) [inline frameworks/kafka/tests/test_sanity.py]: I recommend using time_wait_return() here, to conform with other framework tests: https://github.com/mesosphere/dcos-commons/blob/master/testing/sdk_spin.py#L13

File: frameworks/kafka/tests/test_shakedown.py [hunk 3]
```diff
@@ -237,14 +235,172 @@ def test_plan():
...
+    check_health()
+
+    result = get_connection_info()
+    assert len(result['address']) == 3
+
+    for hostport in result['address']:
+        assert hostport.split(':')[-1] == '9092'
+
+
+# --------- HealtCheck -------------
```
Comment (susanxhuynh) [inline frameworks/kafka/tests/test_shakedown.py]: Typo: missing "h" in "Health"

File: frameworks/kafka/tests/test_shakedown_placement.py [hunk 4]
```diff
@@ -0,0 +1,87 @@
...
+    PACKAGE_NAME,
+    DEFAULT_BROKER_COUNT,
+    service_cli
+)
+
+
+
+def setup_module(module):
+    install.uninstall(PACKAGE_NAME)
+    install.install(PACKAGE_NAME, DEFAULT_BROKER_COUNT)
```
Comment (susanxhuynh) [inline frameworks/kafka/tests/test_shakedown_placement.py]: Is this necessary? Each test below also calls "install()".

File: frameworks/kafka/tests/test_shakedown.py [hunk 5]
```diff
@@ -237,14 +235,172 @@ def test_plan():
...
     tasks = service_cli('state tasks')
     assert tasks
 
 
-# Auxiliary
+# --------- Suppressed -------------
+
+
+@pytest.mark.smoke
+@pytest.mark.smoke
```
Comment (susanxhuynh) [inline frameworks/kafka/tests/test_shakedown.py]: Duplicate line (from above). Can remove.

File: frameworks/kafka/tests/test_shakedown.py [hunk 6]
```diff
@@ -237,14 +235,172 @@ def test_plan():
...
+    def fun():
+        response = dcos.http.get(suppressed_url)
+        response.raise_for_status()
+        return response.text == "true"
+
+    shakedown.wait_for(fun)
+
+
+
+# --------- Port -------------
```
Comment (susanxhuynh) [inline frameworks/kafka/tests/test_shakedown.py]: All of these "Port" tests are duplicated below in test_shakedown_ports.py. Please remove one of the duplicates.

**Comment by mbdcos** [comment]
@susanxhuynh all your comments are addressed

**Comment by susanxhuynh** [comment]
LGTM!

**Comment by mbdcos** [comment]
dropping :  
        [SDK] Fix if STUB_UNIVERSE_URL given, do not build

**Comment by mbdcos** [comment]
Adding test_broker_port.py

**Comment by mbdcos** [comment]
retest this please

**Pull request merged**